UnityProject/Assets/Ardity/Scripts/SerialController.cs

```diff
@@ -63,10 +68,11 @@ public class SerialController : MonoBehaviour
     // ------------------------------------------------------------------------
     void OnEnable()
     {
-        serialThread = new SerialThreadLines(portName, 
-                                             baudRate, 
-                                             reconnectionDelay,
-                                             maxUnreadMessages);
+        serialThread = new SerialThreadLines(portName,
+            baudRate,
+            reconnectionDelay,
+            maxUnreadMessages,
+            dropOldMessage);
```

Comment:
This change on indentation seems to have been automatically done by your tool, but still, I always remove this kind of unintended changes as that doesn't follow the style of the rest of the project.

Reply:
Inverted